### DISCUSSION TITLE
re-edit publii site without backup

### DISCUSSION THREAD

**Discussion by epter80** [GetPublii/Publii#1480] [Help]
"I have an old Publii website but I didn't make a backup and no longer have access to my hosting. However, I do have all the files on my laptop. Can I restore my old website on a new computer without a backup? I've installed Publii on my new computer, but I can't find my old website. Even when I re-upload the files and connect Publii to the new hosting, it doesn't appear. What steps should I take to recover my website?"

**Comment by candidexmedia** [comment]
> However, I do have all the files on my laptop.

Which files do you have? Does the folder contain an input folder and db.sqlite file?

**Comment by epter80** [comment]
hI, only got this. 

![image](https://github.com/GetPublii/Publii/assets/166948065/81d9c7d9-d1d6-4bcc-874e-3f9257667186)

**Reply by candidexmedia** [reply]
This looks like the output folder. Typically, the `input`, `output`, and `preview` folders are stored in the same parent folder like this:

![](https://i.imgur.com/Qt6yEf4.png)

The `input` folder is what's needed to edit the site in Publii. The `output` is what gets published on your server.

Please do a search of your directory, and look for the input folder or its files. It should look something like this:

![](https://i.imgur.com/d7XmzGp.png)

**Comment by epter80** [comment]
it looks like another file structure [candidexmedia](https://github.com/candidexmedia)

**Comment by internettips** [comment]
@epter80 

To confirm, as @candidexmedia suggested your screenshot looks like it's showing the contents of the Publii Output folder. To recover a Publii site more quickly, what you need, is the Input folder contents -- including the db.sqlite file (the Publii database). Fingers crossed.

**Comment by eyk107** [comment]
I have the same problem and still got the input folder + the db.sqlite. How to go further with these files?

**Reply by candidexmedia** [reply]
See: https://github.com/GetPublii/Publii/discussions/1515#discussioncomment-9509430

**Comment by mackonen** [comment]
Stolen laptop and backups and a site that needs updates. There seriously is no way to recover a site without the database?? I mean Publii can import a WP site but not a Publii site??

**Reply by candidexmedia** [reply]
Really sorry to hear about your stolen laptop. As far as I l know though, no, there isn't. Are you sure you don't have a single backup of your computer files on the cloud somewhere? How did you backup the rest of your laptop's files?

> There seriously is no way to recover a site without the database??

You need the entire input folder (which includes the db and config settings) or a backup .tar file.

>  I mean Publii can import a WP site but not a Publii site??

Publii can import a WP site's backup file, which can only be acquired by [logging into the dashboard and generating an export](https://getpublii.com/docs/import-wordpress-into-static-html-site.html). Publii can generate and import its own backups as well.

One of the Publii devs mentioned a potential solution using your sitemap.XML [in another post](https://github.com/GetPublii/Publii/discussions/1515#discussioncomment-9628301), but your theme settings won't be recovered this way.

**Reply by michaelfluegge** [reply]
@mackonen , there are several posts/requests regarding your question. Try a search in this forum and you'll find explanations why a recovery w/o a db is not possible.
